runtime/src/main/java/org/corfudb/security/sasl/plaintext/PlainTextSaslNettyClient.java

```diff
@@ -4,10 +4,14 @@
 import io.netty.channel.ChannelDuplexHandler;
 import io.netty.channel.ChannelHandlerContext;
 
+import io.netty.channel.ChannelPromise;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'io.netty.channel.ChannelPromise'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)